### PR TITLE
create endpoint to add reviews

### PR DESCRIPTION
**Pull request by aladdin91**
# Description
create endpoint to add reviews

# Jira task link
https://hackyourfuture-dk.atlassian.net/browse/VIEW-69

# Screenshots (if applicable)

### CODE REVIEW

File: packages/server/api/controllers/reviews.controller.js
Comment (magdazelena) [inline packages/server/api/controllers/reviews.controller.js:111]: no need to return review if you are not going to use it :)

File: packages/server/api/routes/reviews.router.js
Comment (magdazelena) [inline packages/server/api/routes/reviews.router.js:75]: I am unsure how your express error catching is configured, but I am pretty sure that although in controller you throw a bunch of 400s in specific cases, your rest api will only throw this 500 no matter happens :)

Depending how the express error handling is configures, here you probably should include something like `res.status(error.status).json(({error: error.message})`

File: packages/server/api/controllers/reviews.controller.js
Comment (magdazelena) [reply packages/server/api/controllers/reviews.controller.js:111]: ah, I see. Normally you don't return from inside of try clause, you return after the block.

**Review by magdazelena — CHANGES_REQUESTED** [review]

**Comment by aladdin91** [comment]
Thanks for the review, i will wait for your further review to make my life even more complicated 😆

File: packages/server/api/controllers/reviews.controller.js
Comment (magdazelena) [inline packages/server/api/controllers/reviews.controller.js]: almost there! 

What I would suggest doing would be discarding try/catch at all here (you are caching in router anyway) and plainly do:
```
return knex('reviews').insert(....)
```
at the and of the function. Note I also omitted the await as the function is promise anyway :)

Let me know if that makes sense!
And please, if you are waiting for specifically my feedback re-request review from me :) It's easier for me to not get lost in notifications and reply faster :)

Comment (aladdin91) [reply packages/server/api/controllers/reviews.controller.js]: yes it make sense, so removing the try/catch since we handle it in router and add return to handle the returned promise

**Review by magdazelena — APPROVED** [review]
🎊 🌟

**Pull request closed without merging**
